google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -156,11 +155,9 @@ public static ListOption pageSize(int pageSize) {
   }
 
   /**
-   * If this is for a partitioned read and query and this field is set to `true`, the request will
-   * be executed via Spanner independent compute resources. The method is available in Beta mode
-   * (and is not generally available now).
+   * If this is for PartitionedRead or PartitionedQuery and this field is set to `true`, the request will be
+   * executed via Spanner independent compute resources.
    */
-  @BetaApi
   public static DataBoostQueryOption dataBoostEnabled(Boolean dataBoostEnabled) {
```

Comment:
```suggestion
  public static ReadAndQueryOption dataBoostEnabled(Boolean dataBoostEnabled) {
```
As we are removing the BetaApi annotation now, can we change DataBoostQueryOption to ReadAndQueryOption (similar to how it's for other options)? I understand it's technically a breaking change but I feel as it was a BetaApi, changes may be acceptable (without a major version bump). WDYT, @olavloite ?

Reply:
No, I would not be in favor of that. As you say; this is a breaking change, and it could break existing software, however new it might be. I especially worry about integrations with Beam and similar that often dynamically load other dependencies, such as Cloud Spanner, and that these might break in unexpected ways if we change this now.

Reply:
Makes sense!